client/src/components/DropZone.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Box, Text, Button, ResponsiveContext } from 'grommet';
+import Dropzone from 'react-dropzone'
+
+export default ({
+  fileTypes = [],
+  multiple = true,
+  onDrop,
+}) => {
+  const size = React.useContext(ResponsiveContext)
+  return (
+    <Dropzone onDrop={acceptedFiles => console.log(acceptedFiles)}>
```

Comment:
Should this call the given `onDrop` function?

Reply:
Good call, I updated this and the example!